homeassistant/components/cisco_webex_teams/notify.py

```diff
@@ -51,7 +65,14 @@ def send_message(self, message="", **kwargs):
             title = "{}{}".format(kwargs.get(ATTR_TITLE), "<br>")
 
         try:
-            self.client.messages.create(roomId=self.room, html=f"{title}{message}")
+            if "@" in self.room_or_email:
```

Comment:
Is there no chance that a room ID might have an `@` in it? Might be better to pass both room_id and email down here and send to either or both if not None. 

That way you are adding the flexibility for someone to post to both a room and a 1-1 message. Might be useful for some.

Reply:
First I thought that too, but I didn't want to change too much.
@ is impossible. The room id is always base64 encoded

Reply:
I've modified the code now to allow both parameters at the same time.